### DISCUSSION TITLE
Using catch default skin feels like playing with a semi-HD mod

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#20444] [Ideas]
The outer white layer of the default catch skin fades away instead of being solid "fruit". This gives me a slight impression I'm still playing the Hidden Mod even though I'm playing No Mod.

https://user-images.githubusercontent.com/82471957/192078561-9a4ccde4-7e95-4555-8955-51bba2cb306b.mp4

This is what happens with argon (2022) and triangles (2017) skin:
![image](https://user-images.githubusercontent.com/82471957/192078367-ca2ba9ee-fccd-43b7-a02a-d8a8bc52c527.png)
![image](https://user-images.githubusercontent.com/82471957/192078370-201e57c4-30f2-4c29-a946-a4dda64854ca.png)

Compared to what the classic (2013) skin does:
![image](https://user-images.githubusercontent.com/82471957/192078447-86acda1b-bf93-4fc5-bcd1-20605a39d5e3.png)
![image](https://user-images.githubusercontent.com/82471957/192078448-f6fd87b1-86c8-4cc1-9746-a1aba243f3b2.png)

This is quite a **big** difference of what catch players are used to in stable. Furthermore, adding the Flashlight mod makes it even harder to play compared to what stable does. I suggest removing the fade-out when the fruit is approaching the catcher and make it solid.

I'm aware the default argon skin for catch is still a WIP, so I hope this is something we are considering when making the new default one. Thank you for reading! c:

**Comment by peppy** [comment]
The skin is changing soon. Triangles version will be staying how it is, I think.
